client/configuration/eslint/eslint.js

```diff
@@ -21,13 +21,7 @@ module.exports = {
     "react/jsx-filename-extension": "off",
     "comma-dangle": "off",
     "no-underscore-dangle": "off",
-    quotes: ["error", "double"],
```

Comment:
general comment for future PRs - independent changes like this should be in their own PR.